configure.ac

```diff
@@ -5627,6 +5628,38 @@ AM_CONDITIONAL([ENABLE_HYBRID_SUSPEND], [test x$enable_hybrid_suspend != xno])
 
 dnl End of thread suspend policy
 
+dnl *******************************
+dnl *** Copy stack data support ***
+dnl *******************************
+
+dnl Disable copy_stack_data on platforms supporting full context and
+dnl having access to full stack of running threads.
+if test x$enable_copy_stack_data_default != xno; then
+	case $HOST in
+	X86 | AMD64)
```

Comment:
It is more of Linux/Windows/Android/Wasm thing, or the combinations, eh?

Reply:
This follows the same platforms as default for hybrid suspend. For this case copy stack data will be disabled by default for x86/amd64 linux, osx or windows hosts (note shouldn't include Android, WASM or IOS platforms). It always possible to force it using --enable-copy-stack-data configure parameter.

Reply:
Android runs on x86 too. That is the thing.
Windows runs on arm and arm64, etc. (albeit Mono hasn't been ported to them).
It is confusing processor architecture for other things.

Reply:
Windows is fine since we only have x86, amd64 ports at the moment and if we get more there are things that needs to be updated elsewhere. Android shouldn't have $host_linux set as part if its configuration step, so this should not trigger on Android.